construct.yaml

```diff
@@ -1,22 +1,23 @@
 # name and version (required)
 name: python-microscopy
-version: 19.12.17
+version: 20.07.10
 
 # channels to pull packages from
 channels: &id1
-  - http://repo.anaconda.com/pkgs/main/
-  - http://repo.anaconda.com/pkgs/free/
+  - https://repo.anaconda.com/pkgs/main/
+  - https://repo.anaconda.com/pkgs/free/
   - https://conda.anaconda.org/david_baddeley
-  - http://repo.anaconda.com/pkgs/msys2
+  - https://repo.anaconda.com/pkgs/msys2
 
 # specifications
 specs:
-  - python
+  # - python
+  - pyme-depends
```

Comment:
is a dep of python-microscopy - should not be needed.

Reply:
Fair enough. We'll delete that, too.